php.ini-development

```diff
@@ -1919,6 +1920,11 @@ ldap.max_links = -1
 ; absolute path.
 ;curl.cainfo =
 
+; Define the User-Agent when using cURL for requests. PHP's default setting for this is empty.
+; See user_agent (without prefix) to set the User-Agent when using the HTTP wrapper (e.g. file_get_contents)
+; https://php.net/curl-user-agent
```

Comment:
```suggestion
```

I don't see a need to include this link - `curl.cainfo` doesn't have a link either, and the link might not ever be created.